ws-nextjs-app/pages/[service]/send/[id]/SuccessMessage/index.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ThemeProvider from '#app/components/ThemeProvider';
+import SuccessMessage from './index';
+
+const sampleSuccessResponse = {
+  code: 'OK',
+  submissionId: '269cb7fa-6221-4586-b54f-61c7beda705e',
+  message: 'Success',
+};
+
+const Component = () => (
+  <ThemeProvider service="pidgin">
```

Comment:
Don't think we need the ThemeProvider here as it should be applied globally by Storybook, unless you need to overwrite it for this story?